src/MatrixClientPeg.js

```diff
@@ -136,13 +136,17 @@ class MatrixClientPeg {
     }
 
     _createClient(creds: MatrixClientCreds) {
+        // XXX: This is here and as a require because apparently circular dependencies
+        // are just broken in webpack (https://github.com/webpack/webpack/issues/1788)
+        const UserSettingsStore = require('./UserSettingsStore');
```

Comment:
I think this is a problem that has probably been solved by Granular Settings - the new SettingsStore doesn't depend on a MatrixClient

Reply:
oh, even better :)